clients/src/main/java/org/apache/kafka/clients/admin/internals/FenceProducersHandler.java

```diff
@@ -101,7 +101,8 @@ public ApiResult<CoordinatorKey, ProducerIdAndEpoch> handleSingleResponse(
         InitProducerIdResponse response = (InitProducerIdResponse) abstractResponse;
 
         Errors error = Errors.forCode(response.data().errorCode());
-        if (error != Errors.NONE) {
+        // Concurrent Transaction is still a success for fencing old producers
```

Comment:
nit: when you say old producers here, you just mean the fenced producer?

Reply:
yes